crates/bevy_render/src/render_resource/pipeline_cache.rs

```diff
@@ -595,16 +605,41 @@ impl PipelineCache {
     pub fn queue_render_pipeline(
         &self,
         descriptor: RenderPipelineDescriptor,
+    ) -> CachedRenderPipelineId {
+        // SAFETY: We are setting `unchecked` to `false`, that's always safe
+        unsafe { self.queue_render_pipeline_ex(descriptor, false) }
+    }
+
+    /// Like [`Self::queue_render_pipeline()`], but disables runtime checks for the shader.
+    ///
+    /// # Safety
+    ///
+    /// See: [`wgpu::Device::create_shader_module_unchecked()`].
+    pub unsafe fn queue_render_pipeline_unchecked(
+        &self,
+        descriptor: RenderPipelineDescriptor,
+    ) -> CachedRenderPipelineId {
+        self.queue_render_pipeline_ex(descriptor, true)
+    }
+
+    unsafe fn queue_render_pipeline_ex(
```

Comment:
This should also have safety docs as well.

Reply:
What should it say? 👀